### PR TITLE
Small doc cleanups.

### PR DESCRIPTION
**Pull request by anntzer**
## PR Checklist

- [ ] Has pytest style unit tests (and `pytest` passes).
- [ ] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (run `flake8` on changed files to check).
- [ ] New features are documented, with examples if plot related.
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [ ] Conforms to Matplotlib style conventions (install `flake8-docstrings` and run `flake8 --docstring-convention=all`).
- [ ] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [ ] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).

### CODE REVIEW

File: lib/matplotlib/backends/backend_pdf.py [hunk 1]
```diff
@@ -2640,11 +2640,8 @@ def savefig(self, figure=None, **kwargs):
 
         Parameters
         ----------
-        figure : `.Figure` or int, optional
-            Specifies what figure is saved to file. If not specified, the
-            active figure is saved. If a `.Figure` instance is provided, this
-            figure is saved. If an int is specified, the figure instance to
-            save is looked up by number.
+        figure : `.Figure` or int, default: the active figure.
```
Comment (timhoffm) [inline lib/matplotlib/backends/backend_pdf.py:2643]:
```suggestion
        figure : `.Figure` or int, default: the active figure
```

File: lib/matplotlib/backends/backend_pgf.py [hunk 2]
```diff
@@ -1019,11 +1019,8 @@ def savefig(self, figure=None, **kwargs):
 
         Parameters
         ----------
-        figure : `.Figure` or int, optional
-            Specifies what figure is saved to file. If not specified, the
-            active figure is saved. If a `.Figure` instance is provided, this
-            figure is saved. If an int is specified, the figure instance to
-            save is looked up by number.
+        figure : `.Figure` or int, default: the active figure.
```
Comment (timhoffm) [inline lib/matplotlib/backends/backend_pgf.py:1022]:
```suggestion
        figure : `.Figure` or int, default: the active figure
```

**Review by timhoffm — APPROVED** [review]

File: lib/matplotlib/backends/backend_pdf.py (hunk 1 above)
Comment (anntzer) [reply lib/matplotlib/backends/backend_pdf.py:2643]: yup

File: lib/matplotlib/backends/backend_pgf.py (hunk 2 above)
Comment (anntzer) [reply lib/matplotlib/backends/backend_pgf.py]: yup

**Pull request merged**